### DISCUSSION TITLE
Could we add Emmet to the vs code editor

### DISCUSSION THREAD

**Discussion by leapcow** [cssbattle/forum#352] [Ideas]
I just upgraded to plus when I saw that VS code was an editor option but now I see it doesn't include emmet. I played for a bit trying to load it with an arc boost without any luck. would this be possible? has anyone else had luck loading it into the monaco editor?

**Comment by chinchang** [comment]
Will try adding Emmet. Any directions on that would be appreciated  :)

**Reply by leapcow** [reply]
https://www.npmjs.com/package/emmet-monaco-es

this seems to be the page with the info for the plugin

**Reply by chinchang** [reply]
that plugin says:

> Does NOT support Emmet for embed CSS inside HTML / JSX / TSX

@leapcow Do you think Emmet would be much useful if it works on just HTML in CSSBattle?

**Reply by PigeonSoupGit** [reply]
Came to community to find this, Just upgraded to plus, love the sleek upgrade to my cssbattle window, but the basic "intelisence" autocomplete that the basic editor comes with is now gone and a little annoying.
